sound/soc/sof/pcm.c

```diff
@@ -350,8 +350,6 @@ static int sof_pcm_trigger(struct snd_soc_component *component,
 	if (!ret && reset_hw_params) {
 		ret = sof_pcm_stream_free(sdev, substream, spcm, substream->stream,
 					  free_widget_list);
-		if (ret < 0)
-			return ret;
 	}
 
```

Comment:
as i said there is no need for the curly braces here. Also for the signed off by please add your full name in the format
```
Signed-off-by: FirstName LastName <email>

```

No need to create another pull request just the local patch on your side and then just pull with -f .